### PR TITLE
added documentation for Database util

### LINKED ISSUE #3579: Document Database Util to remove already deleted repositories from database
Hello,

I'm getting an `Error from CL_PACKAGE_FACTORY=>LOAD_PACKAGE 1` error from abapGit 1.97.0 on NW 740 when I click on the name of a repository in the Repository list/overview. The package that belongs to this repository has been deleted and I would like to uninstall/remove the repository from abapGit as it's not needed anymore. Clicking on the ABAP package name of the repository in the overview does throw a proper `Object DEVC <package name> doesn't exist` error.

Of course I can recreate an empty package with the same name and uninstall/remove it then but I think this should work without that too. I read the documentation at https://docs.abapgit.org/guide-online-uninstall.html to see if there's another way to remove a repository but doesn't seem so.

Callstack:
grafik

### PR DESCRIPTION
**Pull request by GiantCrocodile**
Fixes #3579

### CODE REVIEW

File: docs/ref-database-util.md [hunk 1]
```diff
@@ -0,0 +1,188 @@
...
+---
+
+The so-called `Database util` is a tool for managing database entries created by abapGit. You can access the tool via the tools icon in the top right corner of the home page of the abapGit report > choose first menu item `Database util`.
+
+It's possible to edit database entries of type
+
+- `USER` and
+- `REPO`.
+
+An `USER` entry does contain meta information like the favorites of an user and its repository configuration e. g. name and email address for Git. This is an example of an `USER` entry:
```
Comment (g-back) [inline docs/ref-database-util.md:14]: Small spelling/grammar suggestions (not a native speaker so others may correct me as well):
```suggestion
A `USER` entry contains meta information like the favorites of a user and their repository configurations e. g. name and email address for Git. This is an example of a `USER` entry:
```

File: docs/ref-database-util.md [hunk 2]
```diff
@@ -0,0 +1,188 @@
+---
+title: Database util
+category: reference
+order: 99
+---
+
+The so-called `Database util` is a tool for managing database entries created by abapGit. You can access the tool via the tools icon in the top right corner of the home page of the abapGit report > choose first menu item `Database util`.
```
Comment (FreHu) [inline docs/ref-database-util.md:7]:
```suggestion
`Database util` is a tool for managing database entries created by abapGit. You can access the tool via the tools icon in the top right corner of the abapGit home page > choose first menu item `Database util`.
```

File: docs/ref-database-util.md [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+    <CODE_INSPECTOR_CHECK_VARIANT>ZMY_CHECK_VARIANT</CODE_INSPECTOR_CHECK_VARIANT>
+    <BLOCK_COMMIT>X</BLOCK_COMMIT>
+    <SERIALIZE_MASTER_LANG_ONLY/>
+   </LOCAL_SETTINGS>
+  </REPO>
+ </asx:values>
+</asx:abap>
+```
+
+If you remove a repository entry from the database util, the repository is not shown in abapGit anymore. This is useful if you want to remove a repository which has a non-existing package assigned and thus can't be clicked on to be select it and delete it. This is like a workaround for such a case.
```
Comment (FreHu) [inline docs/ref-database-util.md:186]:
```suggestion
If you remove a repository entry from the database util, the repository is not shown in abapGit anymore. This is useful for example if you want to remove a repository which has a non-existing package assigned and thus can't be opened and deleted normally.
```

**Comment by mbtools** [comment]
Thanks for the addition to docs. Let’s add a disclaimer at the beginning of the page that the DB Util should only be used in emergency cases when regular abapGit functionality doesn’t work. Also users should report such cases as new issues on github so we can fix the code that caused such situation.

**Comment by GiantCrocodile** [comment]
@mbtools I will change the current disclaimer at the end and improve it a bit in the next days. On the other hand I'm not sure if that disclaimer is not a contradiction to the issue/use case why the documentation was added at all (see comment no. 1 of this PR). So maybe we should have that disclaimer + make sure deletion works even when the defined package doesn't exist anymore.

**Comment by larshp** [comment]
thanks 👍

**Pull request merged**